app/uk/gov/hmrc/helptosavetestadminfrontend/controllers/DeleteVerifiedEmailsController.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosavetestadminfrontend.controllers
+
+import com.google.inject.Inject
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc._
+import uk.gov.hmrc.helptosavetestadminfrontend.config.AppConfig
+import uk.gov.hmrc.helptosavetestadminfrontend.forms.EmailsForm
+import uk.gov.hmrc.helptosavetestadminfrontend.repos.VerifiedEmailMongoRepository
+import uk.gov.hmrc.helptosavetestadminfrontend.views
+
+import scala.concurrent.Future
+
+class DeleteVerifiedEmailsController @Inject()(verifiedEmailRepo: VerifiedEmailMongoRepository)(implicit override val appConfig: AppConfig,
```

Comment:
you can rename this as `EmailController` - controller names generally do not specify actions , but resources

Reply:
Yeah I'm thinking VerifiedEmailController is more suitable

Reply:
ok, thats fine, but the endpoint doesn't say about `verified`